external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -88,8 +84,6 @@ def _get_input_vector(
             tf.keras.layers.Input(shape=[n_features]) for n_features in features
         ]
     packed = tf.keras.layers.Concatenate()(input_layers)
-    if scaler is not None:
-        packed = scaler.normalize_layer(packed)
     return input_layers, packed
```

Comment:
To be honest, I don't see this being a widely-used utility. If I were to move out the concatenation step, the only thing that remains is an `input_layers = [list comprehension]` operation which could be inserted directly into the code instead of calling `get_input_vector`. I shared it in this case to turn two lines of code into one line.

If you think it would be clearer, I can inline this function in `precipitative.py` and return it to `recurrent.py`. In `recurrent.py` it is a lot more important to turn these two lines into one, because this call happens several times.